src/libmongoc/src/mongoc/mongoc-config.h.in

```diff
@@ -381,6 +381,17 @@
 #  undef MONGOC_ENABLE_CLIENT_SIDE_ENCRYPTION
 #endif
 
+
+/*
+ * Set if struct sockaddr_storage has __ss_family (instead of ss_family)
+ */
+
+#define MONGOC_HAVE_SS_FAMILY @MONGOC_HAVE_SS_FAMILY@
+
+#if MONGOC_HAVE_SS_FAMILY != 1
```

Comment:
Apologies, this is a bug with my original patch. But CHECK_STRUCT_HAS_MEMBER may leave MONGOC_HAVE_SS_FAMILY unset if ss_family is not found. If so, then this resolves to:
```
#if != 1
```
Causing [this error](https://evergreen.mongodb.com/task/mongo_c_driver_releng_link_with_bson_mingw_patch_25494023026066a58cde196fac84b2138eae8fac_5de68f5a562343176d5f9142_19_12_03_17_18_01):

```
C:/data/mci/ce4bdea56babc8e3f78543aa3866aff5/mongoc/build-dir/src/libmongoc/src/mongoc/mongoc-config.h:391:27: error: operator '!=' has no left operand
 #if MONGOC_HAVE_SS_FAMILY != 1
                           ^
mingw32-make[2]: *** [src/libmongoc/CMakeFiles/mongoc_shared.dir/src/mongoc/mongoc-aggregate.c.obj] Error 1
```

Can you modify the CmakeLists.txt above to be:
```
CHECK_STRUCT_HAS_MEMBER("struct sockaddr_storage" ss_family "sys/socket.h" MONGOC_HAVE_SS_FAMILY)
if (NOT MONGOC_HAVE_SS_FAMILY)
   set (MONGOC_HAVE_SS_FAMILY 0)
else ()
   set (MONGOC_HAVE_SS_FAMILY 1)
endif ()
```

I believe that should fix the issue, and I will rerun our test suite.

Reply:
Pull request updated.